war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1658,9 +1658,25 @@ function expandTextArea(button,id) {
 // by using the contents fetched from the given URL.
 function refreshPart(id,url) {
     var intervalID = null;
+    var refreshInterval = 5000; //default is 5s
+
     var f = function() {
         if(isPageVisible()) {
             new Ajax.Request(url, {
+                onCreate: function() {
+                    requestStartTime = new Date().getTime();
```

Comment:
Is this variable in scope so that it can be read on `onComplete` ?

Reply:
I tried it locally by changing the if it's working fine. I have no idea on how to slow answer to the Ajax calls, I don't know where this goes on server side.